notion_scholar/bibtex.py

```diff
@@ -37,7 +37,7 @@ def get_bib_database_from_string(string: str) -> BibDatabase:
         string.
     """
     bibtex_parser = BibTexParser(
-        interpolate_strings=False,
+        # interpolate_strings=False,
```

Comment:
Why did you remove the string interpolation ?

Reply:
My bad, I forgot to undo that comment. Will update PR